src/db/schema.ts

```diff
@@ -0,0 +1,146 @@
+import { relations } from "drizzle-orm";
+import {
+  customType,
+  integer,
+  sqliteTable,
+  text,
+} from "drizzle-orm/sqlite-core";
+
+const timestamp = customType<{
+  data: Date;
+  driverData: string;
+}>({
+  dataType() {
+    return "datetime";
+  },
+  fromDriver(value: string): Date {
+    return new Date(value);
+  },
+});
+
+export const sets = sqliteTable("sets", {
+  id: text("id").primaryKey().notNull(),
+  name: text("name"),
+  abbr: text("abbr"),
+  series_id: text("series_id").references(() => series.id),
```

Comment:
So in Drizzle the attribute name will be what the returned object will have, and the first parameter of the Drizzle function is the table name in the DB. Usually TS projects use lowerCamalCase for objects, so you might want to follow that same pattern here if that's the planned naming convention for this project:

```suggestion
  seriesId: text("series_id").references(() => series.id),
```

Reply:
I go against a lot of "JS norms" in my personal style 😅 

`variable_names`
`functionNames`
`CONSTANT_NAMES` (top level consts that act like config, mainly. This one is a lot looser in definition, and I kinda just do it by feel?)
`TypeNames` / `ClassNames`

I am unsure if this can even be enforced at an eslint/prettier level, but would be nice...

Reply:
Can I just resolve this one then?